src/network/postgres/postgres_network_commands.cpp

```diff
@@ -123,6 +123,17 @@ Transition SimpleQueryCommand::Exec(const common::ManagedPointer<ProtocolInterpr
     t_cop->BeginTransaction(connection);
   }
 
+  // Set statements are manually handled here.
```

Comment:
What if one comes across in Extended Query?

Reply:
Added some logic to Bind and Execute.